### ISSUE TITLE
[base_custom_info] Feature request: Properties with several selected values

### ISSUE THREAD

**Issue by diggy128** [OCA/server-tools#891]
Are there any plans to introduce one2many or many2many properties?

**Comment by pedrobaeza** [comment]
No, I think you are mixing concepts. What do you want to achieve?

**Comment by diggy128** [comment]
Maybe I should rephrase:
As I can have values of string, float, many2one type, I want to have one2many values.
eg Have a property Dietary Groups and be able to select Halal AND Kosher AND Vegan as values.

**Comment by pedrobaeza** [comment]
The text is still incorrect. You don't want a One2many. You want a field where you can select more than one value. That's not an intended feature for now because that makes more difficult data-mining, but it can be added if someone contributes it.

**Comment by diggy128** [comment]
After taking a better look at the code, I think what is needed is a field to select multiple options for a value.
In any case, I just started backporting base_custom_info to v8 and I need this feature. I just wanted to check if there is anyone working on this particular feature so as not to duplicate.
In any case once finished porting, I'll create a pull request and then discuss the best way to implement this feature.
I plan to migrate data from base_custom_attributes that used a different approach to EAV so it'll take some time. In any case I found the UI for base_custom_attributes to be way superior than base_custom_info since it essentially created a dynamic view, but got to go with the flow...
